README.txt

```diff
@@ -6,29 +6,38 @@ This folder contains a simulator for an ALTERNATE BIT PROTOCOL implemented in Ca
 
 README.txt	
 
+data [This folder contains the data files for the simulator]
+	input
+		input_abp_0

```

Comment:
All other sections in the README file provides extensions of the file ( .pdf or .hpp or .cpp). To have the README file in the same structure/format, I would suggest to update these files with .txt (example, input_abp_0.txt).

Reply:
it has been fixed